src/PharoThings-Devices-HCSR04.package/PotHCSR04Device.class/instance/readPinStateDuration.st

```diff
@@ -0,0 +1,7 @@
+controlling
+readPinStateDuration
+	| pulseStart pulseEnd |
+	echoPin beDigitalInput; enablePullDownResister.
```

Comment:
My idea was that all such pin configurations should be performed in device>>#connect method and therefore will happens only once when you install device to the board.

But for this you will need to create device with pin instances instead of numbers.

Reply:
Ok. I see that it is different story. Ignore it